src/Kafka/Producer.purs

```diff
@@ -148,22 +148,58 @@ type ProducerBatchImpl =
     , topicMessages :: Array TopicMessagesImpl
     )
 
+-- | see [Options](https://kafka.js.org/docs/producing#options)
+-- |
+-- | * `allowAutoTopicCreation`
+-- |   * Allow topic creation when querying metadata for non-existent topics
+-- |   * default: `true`
+-- | * `idempotent`
+-- |   * If enabled producer will ensure each message is written exactly once. Acks must be set to `-1` ("all"). Retries will default to `MAX_SAFE_INTEGER`.
+-- |   * default: `false`
+-- | * `maxInFlightRequests`
+-- |   * Max number of requests that may be in progress at any time. If falsey then no limit.
```

Comment:
What is falsey when dealing with `Maybe Int`? `Nothing`? Is `Just 0` falsey in this context?

Reply:
Copied from the doc [here](https://kafka.js.org/docs/producing#options).